src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -84,16 +94,22 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
           <ArtworkSidebarPartnerInfoFragmentContainer artwork={artwork} />
         </>
       )}
-      <Join separator={<Spacer mt={2} />}>
-        <AuthenticityCertificateFragmentContainer artwork={artwork} />
-        <SecurePaymentFragmentContainer artwork={artwork} />
-        <VerifiedSellerFragmentContainer artwork={artwork} />
-        <BuyerGuaranteeFragmentContainer artwork={artwork} />
-      </Join>
+
+      {shouldRenderArtworkBadges && (
+        <Join separator={<Spacer mt={2} />}>
+          <Separator mt={2} />
+          <AuthenticityCertificateFragmentContainer artwork={artwork} />
+          <SecurePaymentFragmentContainer artwork={artwork} />
```

Comment:
**question**: Would we hide this "secure payment" component unless there are other badged to render? Is that desired? I'm wondering if the secure payment component should always render.

Reply:
> Would we hide this "secure payment" component unless there are other badged to render?

No, because "secure payment" component  has its own check, which determines whether this component should be rendered or not

Reply:
small clarification: "secure payment" and "buyer guarantee" components do the same checks

Reply:
Fixed